src/app/main/main.html

```diff
@@ -46,6 +46,9 @@
       </div>
     </div>
     <div class="pane vis-list-pane">
+      <welcome-modal>
+        <h1>Welcome to Voyager</h1>
+      </welcome-modal>
```

Comment:
This could be abstracted into its own `<voyager-welcome-modal>` template, but it was easy to make a `<welcome-modal>` directive that encapsulates the localStorage behavior so I went ahead and did so